files/help/ares-package.help

```diff
@@ -5,34 +5,42 @@
     "name" : "ares-package - Create a package containing a webOS app or service",
     "synopsis" : {
         "default" : [
-            "ares-package [OPTION...] APP_DIR [SERVICE_DIR [...]]"
+            "ares-package [OPTION...] APP_DIR [SERVICE_DIR [...]]",
+            "ares-package --info(-i) IPK_PACKAGE",
```

Comment:
Why don't you replace "IPK_PACKAGE" to PACKAGE_FILE? 
ares-package --info(-i) <PACKAGE_FILE>

Please refer to ares-install help.

Reply:
I've updated this.